less/variables.less

```diff
@@ -23,4 +23,21 @@
 @btn-bg-color: #22283A;
 @border-real: #dee2e6;
 
+// font-size
+@name: 40px;
+@nav: 16px;
+@button: 18px;
+
+// padding
+@smallpadding: 10px;
+@mediumpadding: 20px;
+@largepadding: 30px;
+
+// margin
+@smallmargin: 10px;
+@mediummargin: 20px;
+@largemargin: 30px;
```

Comment:
beautiful variables buuuuuut!! the px messed them up. please use rems or ems instead. For example, 5px will be .5rem if the html is set to 62.5% (which is 10px) etc